samples-aspnetcore-3x/self-hosted-login/okta-aspnetcore-mvc-example/Startup.cs

```diff
@@ -62,6 +84,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
+            app.UseMiddleware<OktaMiddleware>();
```

Comment:
What does this middleware do?

Reply:
This was my approach to ensure that the user is set from the session if it's there; there may be a better way to do this: https://github.com/okta/samples-aspnetcore/pull/51/files#diff-ca2b1670d6af7911c1b6e97569b6f8da41772bf6a5a3e288514f1cf6160071ecR27